astropy/time/src/parse_times.c

```diff
@@ -1,32 +1,32 @@
+#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
+#include "Python.h"
+#include "numpy/arrayobject.h"
+#include "numpy/ufuncobject.h"
 #include <stdio.h>
 #include <string.h>
 
+#define MODULE_DOCSTRING \
+    "Fast time parsers.\n\n" \
+    "This module allows one to create gufuncs that vectorize the parsing of\n" \
+    "standard time strings."
+#define CREATE_PARSER_DOCSTRING \
+    "create_parser()\n\n" \
+    "Create a gufunc that parsers according to the passed in parameters.\n\n" \
+    "Parameters\n" \
+    "----------\n" \
+    "pars : ~numpy.ndarray\n" \
+    "    Should be structured array with delim, start, stop, break_allowed for each\n" \
+    "    of year, month, day, hour, minute, integer second, fractional second.\n\n" \
+    "Returns\n" \
+    "-------\n" \
+    "parser : `~numpy.ufunc`\n" \
+    "    Will parse bytes or unicode."
```

Comment:
Can you clarify what's meant by unicode here? I guess this means numpy UTF4, or ?

Reply:
Hmm, that is really a leftover of what I wanted to happen.  Right now, it would be possible to extent to UTF4, but little point until unicode is more directly supported for ufuncs. I replaced this text with 'Suitable for use by `~astropy.time.TimeString` formats.'